python/lsst/sims/maf/bundles/__init__.py

```diff
@@ -0,0 +1 @@
+from .glanceBundle import *
```

Comment:
I think I like the idea of adding these bundles here, at least in theory (the idea being that then it's easier to import into multiple scripts or at the command line?). 
Why not drag the bundles out of sciencePerformance and schedulerValidation at the same time?